src/modules/shared/components/ShareFolderBlade/ShareFolderBlade.tsx

```diff
@@ -55,17 +54,10 @@ const ShareFolderBlade: FC<ShareFolderBladeProps> = ({ isOpen, onClose, folderId
 	const handleSend = async () => {
 		try {
 			if (user) {
-				await CampaignMonitorService.send({
-					template: EmailTemplate.SHARE_FOLDER,
+				await foldersService.shareFolder(folderId, {
 					data: {
 						to: emailInputValue,
 						consentToTrack: 'unchanged',
-						data: {
-							sharer_email: user.email,
-							sharer_name: user.fullName,
-							folder_name: folderName,
-							folder_sharelink: link,
-						},
 					},
 				});
```

Comment:
This is a bit of a strange signature for this function
why isn't it just:
```
await foldersService.shareFolder(folderId, emailInputValue);
```
since the consertToTrack isn't needed anymore since that is part of the internal logic of the endpoint in the backend